shared/src/CommonClientResponseCodes.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Azure.Devices.Shared
+{
+    /// <summary>
+    /// A list of common status codes to represent the response from the client.
+    /// </summary>
+    /// <remarks>
+    /// These status codes are based on the HTTP status codes listed here <see href="http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml"/>.
+    /// </remarks>
+    [SuppressMessage("Design", "CA1052:Static holder types should be Static or NotInheritable",
```

Comment:
Seems low value to me. Did you consult with Azure SDK team?

Reply:
I haven't, I'll ask if they have a recommendation.